docs/src/pages/demos/tables/EnhancedTable.js

```diff
@@ -148,6 +154,118 @@ EnhancedTableToolbar.propTypes = {
 
 EnhancedTableToolbar = withStyles(toolbarStyles)(EnhancedTableToolbar);
 
+const footerStyles = theme => ({
+  cell: {
+    padding: '0 !important',
+  },
+  toolbar: {
+    height: 56,
+    minHeight: 56,
+    paddingRight: 2,
+  },
+  spacer: {
+    flex: '1 1 100%',
+  },
+  select: {
+    marginLeft: 8,
+    width: 34,
+    textAlign: 'right',
+    paddingRight: 22,
+    color: theme.palette.text.secondary,
+    height: 32,
+    lineHeight: '32px',
+  },
+  selectRoot: {
+    marginRight: 32,
+  },
+  actions: {
+    color: theme.palette.text.secondary,
+    marginLeft: 20,
+  },
+  title: {
+    flex: '0 0 auto',
+  },
+});
+
+let EnhancedTableFooter = props => {
```

Comment:
It's the first introduction of an anonymous arrow function, please stick to the `function` convention we use.

Reply:
It isn't, look at `EnhancedTableToolbar`. :wink:

Reply:
🙈  oh noww entropyyyy -sum p*log(p)

Reply:
Don't worry, I'll change both.

Reply:
I know why this is an anonymous arrow function! To apply the styles, the variable is re-assigned, which throws a linter error (no-func-assign) if they are functions.
What to do now?